scania-digital-design-system/sdds
# feat(utilities/colour): migrate utilities/colours

**Describe pull-request**  
Migrate the utilities/colour story.

**Solving issue**  
Fixes: AB#2481

**How to test**  
1. Go to storybook link below
2. Check in Utilities -> Colours
3. Check that the correct colors are being displayed.

**Screenshots**  
-

**Additional context**  
-
